Makefile

```diff
@@ -59,163 +34,230 @@ ifeq ("$(DEBUG_BUILD)", "true")
+		--tag $(BUILDAH_TRANSPORT)$(PGO_IMAGE_PREFIX)/$(PGO_IMAGE_NAME):$(PGO_IMAGE_TAG) \
+		--label name='$(PGO_IMAGE_NAME)' \
+		--label build-date='$(PGO_IMAGE_TIMESTAMP)' \
+		--label description='$(PGO_IMAGE_DESCRIPTION)' \
+		--label maintainer='$(PGO_IMAGE_MAINTAINER)' \
+		--label summary='$(PGO_IMAGE_SUMMARY)' \
+		--label url='$(PGO_IMAGE_URL)' \
+		--label vcs-ref='$(PGO_IMAGE_REVISION)' \
+		--label vendor='$(PGO_IMAGE_MAINTAINER)' \
+		--label io.k8s.display-name='$(PGO_IMAGE_NAME)' \
+		--label io.k8s.description='$(PGO_IMAGE_DESCRIPTION)' \
+		--label io.openshift.tags="postgresql,postgres,sql,nosql,crunchy" \
+		--annotation org.opencontainers.image.authors='$(PGO_IMAGE_MAINTAINER)' \
+		--annotation org.opencontainers.image.vendor='$(PGO_IMAGE_MAINTAINER)' \
+		--annotation org.opencontainers.image.created='$(PGO_IMAGE_TIMESTAMP)' \
+		--annotation org.opencontainers.image.description='$(PGO_IMAGE_DESCRIPTION)' \
+		--annotation org.opencontainers.image.revision='$(PGO_IMAGE_REVISION)' \
+		--annotation org.opencontainers.image.title='$(PGO_IMAGE_SUMMARY)' \
+		--annotation org.opencontainers.image.url='$(PGO_IMAGE_URL)' \
+		$(if $(PGO_VERSION),$(strip \
+			--label release='$(PGO_VERSION)' \
+			--label version='$(PGO_VERSION)' \
+			--annotation org.opencontainers.image.version='$(PGO_VERSION)' \
+		)) \
+		--file $< --format docker --layers .
+
+##@ Test
 .PHONY: check
-check:
+check: ## Run basic go tests with coverage output
 	$(GO_TEST) -cover ./...
 
 # Available versions: curl -s 'https://storage.googleapis.com/kubebuilder-tools/' | grep -o '<Key>[^<]*</Key>'
 # - KUBEBUILDER_ATTACH_CONTROL_PLANE_OUTPUT=true
 .PHONY: check-envtest
+check-envtest: ## Run check using envtest and a mock kube api
 check-envtest: ENVTEST_USE = hack/tools/setup-envtest --bin-dir=$(CURDIR)/hack/tools/envtest use $(ENVTEST_K8S_VERSION)
 check-envtest: SHELL = bash
 check-envtest:
 	GOBIN='$(CURDIR)/hack/tools' $(GO) install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
 	@$(ENVTEST_USE) --print=overview && echo
 	source <($(ENVTEST_USE) --print=env) && PGO_NAMESPACE="postgres-operator" $(GO_TEST) -count=1 -cover -tags=envtest ./...
 
-# - PGO_TEST_TIMEOUT_SCALE=1
+# The "PGO_TEST_TIMEOUT_SCALE" environment variable (default: 1) can be set to a
+# positive number that extends test timeouts. The following runs tests with 
+# timeouts that are 20% longer than normal:
+# make check-envtest-existing PGO_TEST_TIMEOUT_SCALE=1.2
 .PHONY: check-envtest-existing
+check-envtest-existing: ## Run check using envtest and an existing kube api
 check-envtest-existing: createnamespaces
 	${PGO_KUBE_CLIENT} apply --server-side -k ./config/dev
 	USE_EXISTING_CLUSTER=true PGO_NAMESPACE="postgres-operator" $(GO_TEST) -count=1 -cover -p=1 -tags=envtest ./...
 	${PGO_KUBE_CLIENT} delete -k ./config/dev
 
 # Expects operator to be running
 .PHONY: check-kuttl
-check-kuttl:
+check-kuttl: ## Run kuttl end-to-end tests
 	${PGO_KUBE_CLIENT} ${KUTTL_TEST} \
```

Comment:
🌱 I would not mind if this became (only) `${KUTTL_TEST}` with the variable defined this way:

```make
KUTTL ?= kubectl-kuttl
KUTTL_TEST ?= $(KUTTL) test
```

One of my (older, I think...) k3s VMs does a wonky thing where KUTTL invoked as a plugin by `kubectl` resolves `sh` to `dash` (inside the k3s files) which can't run our tests due to Bash-isms. 😖 

We can focus a test or set flags lots of ways; this is one:
`make check-kuttl KUTTL_TEST='$(KUTTL) test --timeout=xyz'`